src/libsync/propagateremotemkdir.cpp

```diff
@@ -127,6 +126,10 @@ void PropagateRemoteMkdir::propfindResult(const QVariantMap &result)
     }
     if (result.contains("id")) {
         _item->_fileId = result["id"].toByteArray();
+        _item->_fileIdLocal = result["fileid"].toByteArray();
+        if (_item->_fileIdLocal.isEmpty()) {
+            _item->_fileIdLocal = _item->_fileId.left(8);
```

Comment:
Should all the .left(8) magic (it's in other commits too right?) be moved to a function that has a fat comment to remove the whole stuff in a year or two?